src/index.html

```diff
@@ -12,7 +13,49 @@
       href="./style.css"
     />
   </head>
+  <!--#endregion-->
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```

Comment:
HTML Content: It would be better to use semantic tags instead of `div` for the `stars` blocks. Consider using a semantic tag like `section` or `article` that would better describe the content.